src/get-config.c

```diff
@@ -0,0 +1,44 @@
+/* 
+ * Copyright (c) 2017, SingularityWare, LLC. All rights reserved.
+ * Copyright (c) 2015-2017, Gregory M. Kurtzer. All rights reserved.
+ * Copyright (c) 2017, Vanessa Sochat All rights reserved.
```

Comment:
it's my first function in C! Ever! I'm so proud. It's only taking me many many hours to write like, 10 lines :)